Introduction to xemc3.ipynb

```diff
@@ -0,0 +1,3171 @@
+       "       [0.349066, 0.366519],\n",
+       "       [0.366519, 0.383972],\n",
+       "       [0.383972, 0.401426],\n",
+       "       [0.401426, 0.418879],\n",
+       "       [0.418879, 0.436332],\n",
+       "       [0.436332, 0.453786],\n",
+       "       [0.453786, 0.471239],\n",
+       "       [0.471239, 0.488692],\n",
+       "       [0.488692, 0.506145],\n",
+       "       [0.506145, 0.523599],\n",
+       "       [0.523599, 0.541052],\n",
+       "       [0.541052, 0.558505],\n",
+       "       [0.558505, 0.575959],\n",
+       "       [0.575959, 0.593412],\n",
+       "       [0.593412, 0.610865],\n",
+       "       [0.610865, 0.628319]])\n",
+       "Coordinates:\n",
+       "    phi_bounds  (phi, delta_phi) float64 0.0 0.01745 0.01745 ... 0.6109 0.6283\n",
+       "Dimensions without coordinates: phi, delta_phi"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "ds.coords['phi_bounds']"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a9c4002f",
+   "metadata": {},
+   "source": [
+    "## ds.emc3.plot_Rz(parameter, phi = $\\phi$)\n",
+    "\n",
+    "The parameter is given as a string, and the valid parameters is the same as the measurements that is possible to do in W7X. The angle phi $= \\phi$ is the angle given in radians as floats. \n",
+    "\n",
+    "The floats of the angle $\\phi$ can be found in the dictionary defined by ds.coords['phi_bounds'] which has 2 dimensions; one for the actual angles $\\phi$, and another for the change in angle $d\\phi$. There are 36 different values for $\\phi$ since the reactor has a five-fold symmetry: $5\\cdot 36 = 180^{\\circ}$.\n",
+    "\n",
+    "In the cells below are some examples of the parameter electron temperature $T_e$ plotted in toroidal slices for phi index $n_{\\phi} = [0,18,35]$."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "1a8d6533",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "fb327ea090b14f29a83ee0b93c276153",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "interactive(children=(IntSlider(value=0, description='iy', max=35), Output()), _dom_classes=('widget-interact'…"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "text/plain": [
+       "<function __main__.plot_Te(iy)>"
+      ]
+     },
+     "execution_count": 11,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import ipywidgets as widgets\n",
+    "from ipywidgets import interact, interactive, fixed, interact_manual\n",
+    "def plot_Te(iy):\n",
+    "    plt.figure(figsize=(20,10))\n",
+    "    ds.emc3.plot_Rz(\"Te\", phi=ds.coords['phi_bounds'][iy][0])\n",
+    "iy = widgets.IntSlider(min = 0, max = 35, value = 0)\n",
```

Comment:
why not define a realSlider and pass directly a float?
plot_Rz is not restricted to the resolution of the data.